docs/spec/parallel_dask.rst

```diff
@@ -0,0 +1,60 @@
+.. _parallel_dask:
+
+============================
+Parallel Execution with Dask
+============================
+
+
+Queries
+-------
+
+Kartothek executes queries with a task for every parquet file to read. These
+files are filtered by the predicate's restrictions on partitions and indexed
+columns.
+
+.. image:: /images/kartothek_read_dispatch.png
+  :width: 600
+  :alt: Kartothek dispatches tasks by parquet file
+
+
+For every file its range of values (stated in the parquet footer) is matched
+against the query. (See :doc:`/spec/predicate_pushdown`)
```

Comment:
You also removed the `/spec/predicate_pushdown` page in this PR. If we link it here it should exist

Reply:
Or we can explain predicate pushdown by itself in "Efficient Querying" and point this link to that section